CHANGES.rst

```diff
@@ -54,6 +54,11 @@ Major changes
   compliance with the scikit-learn API.
   :pr:`647` by :user:`Guillaume Lemaitre <glemaitre>`
 
+* Fixes a bug in :class:`TableVectorizer` where `remainder` is now cloned if this is
+  a transformer such that the same instance is not shared between different
+  transformers.
```

Comment:
```suggestion
* Fixes a bug in :class:`TableVectorizer` with `remainder`: it is now cloned if it's
  a transformer so that the same instance is not shared between different
  transformers.
```